src/e2e-test/features/gcs/sink/GCSSink.feature

```diff
@@ -199,3 +199,69 @@ Feature: GCS sink - Verification of GCS Sink plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify data is transferred to target GCS bucket
+
+  @BQ_SOURCE_DATATYPE_TEST @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred successfully from BigQuery to GCS with supported datatypes
```

Comment:
nit: `To verify data is getting transferred successfully from BigQuery to GCS using json output format`

Reply:
Change done.